source/patterns/@aws-solutions-constructs/aws-lambda-kendra/lib/index.ts

```diff
@@ -0,0 +1,197 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+// Imports
+import * as defaults from "@aws-solutions-constructs/core";
+import * as lambda from "aws-cdk-lib/aws-lambda";
+import * as iam from "aws-cdk-lib/aws-iam";
+import * as kendra from "aws-cdk-lib/aws-kendra";
+import * as ec2 from "aws-cdk-lib/aws-ec2";
+import { Construct } from "constructs";
+
+/**
+ * @summary The properties for the LambdaToKendra class.
+ */
+export interface LambdaToKendraProps {
+  /**
+   *
+   *
+   * @default - Optional user provided props to override the default props for the Kendra index. Is this required?
```

Comment:
Similar comment to the one in the README. Feels strange to end the comment in a question about this being required.